lib/compress/zstd_compress_internal.h

```diff
@@ -81,6 +81,25 @@ typedef struct {
     ZSTD_fseCTables_t fse;
 } ZSTD_entropyCTables_t;
 
+typedef struct {
```

Comment:
Try to define and document "metadata" in this context, as its a fairly generic term.

Reply:
Sounds good!